### ISSUE TITLE
Main Exception Message: unknown error: Chrome failed to start: exited abnormally.

### ISSUE THREAD

**Issue by bibhubal** [anilabhadatta/educative.io_scraper#36]
Main Exception Message: unknown error: Chrome failed to start: exited abnormally.

  (unknown error: DevToolsActivePort file doesn't exist)

  (The process started from chrome location /home/osboxes/Desktop/educative.io_scraper-master/Chrome-bin/linux/chrome/chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)

Stacktrace:

#0 0x5576c832eb33 <unknown>

#1 0x5576c7df76d8 <unknown>

#2 0x5576c7e1a84c <unknown>

#3 0x5576c7e15fca <unknown>

#4 0x5576c7e50e0a <unknown>

#5 0x5576c7e4af53 <unknown>

#6 0x5576c7e20a0a <unknown>

#7 0x5576c7e21ad5 <unknown>

#8 0x5576c83602fd <unknown>

#9 0x5576c83794bb <unknown>

#10 0x5576c83620d5 <unknown>

#11 0x5576c837a145 <unknown>

#12 0x5576c8355aaf <unknown>

#13 0x5576c8396ba8 <unknown>

#14 0x5576c8396d28 <unknown>

#15 0x5576c83b148d <unknown>

#16 0x7f3895f57b43 <unknown>

**Comment by anilabhadatta** [comment]
@bibhubal make a video by doing a fresh start and send here

**Comment by bibhubal** [comment]
is there any issues with --log-level=3

**Comment by bibhubal** [comment]
@anilabhadatta Please find the vidio

https://user-images.githubusercontent.com/9507639/224297874-da211b38-e453-40b0-9844-d87bef47c08d.mp4

Please find the vidio .
I am using ubuntu 22.04

**Comment by anilabhadatta** [comment]
@bibhubal You don't have to copy anything from Chrome driver folder.
Download the chrome binary from releases in my repo for linux amd64 and paste it inside Chrome bin folder.
Folder Structure Chrome-bin/linux/chrome/chrome(exe)

**Comment by anilabhadatta** [comment]
@bibhubal 
Also no need to install virtualenv and create env everytime.
That process is required only once.
Also i would suggest not to change the config, let the default configuration be 0

**Comment by bibhubal** [comment]
@anilabhadatta 
Hi I have tried again by freshly installing ubuntu OS followed the exact procedure but again same issue.
This is completely Fresh setup from scratch. New latest Ubuntu OS machine installed in virtual box. 

Starting ChromeDriver 98.0.4758.102 (273bf7ac8c909cde36982d27f66f3c70846a3718-refs/branch-heads/4758@{#1151}) on port 9515

Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
Starting ChromeDriver 98.0.4758.102 (273bf7ac8c909cde36982d27f66f3c70846a3718-refs/branch-heads/4758@{#1151}) on port 9515
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.

 **Invalid --log-level value.
Unable to initialize logging. Exiting...**

Not sure why chrome is exiting when i am trying to go for option 3 which is" Press 3 and Enter to login Educative"

**Comment by anilabhadatta** [comment]
@bibhubal send a video again

**Comment by bibhubal** [comment]
@anilabhadatta 
[Screencast from 2023-03-10 11-40-32.webm](https://user-images.githubusercontent.com/9507639/224375011-757794ce-5805-403c-a82b-90e0d59a86ad.webm)

**Comment by anilabhadatta** [comment]
@bibhubal You are not reading the readme properly, Chromedriver and Chrome are two separate entities.
Please do not copy chromedriver to any other location.
https://github.com/anilabhadatta/educative.io_scraper/releases/download/Educative-Scraper-v6.8/Chrome-bin-linux-amd64.zip
Download this zip file, it contains the Chrome binary for educative scraper.

**Comment by bibhubal** [comment]
Thanks @anilabhadatta My Bad I almost Forgot about bin files in release sections. Now i am closing this issue . But there is another issue i will raise it in another thread . Thanx.
